src/topology/instances/ennreal.lean

```diff
@@ -724,6 +724,21 @@ begin
   exact exists_congr (λ hr, nnreal.has_sum_iff_tendsto_nat)
 end
 
+lemma nnreal.has_sum_of_real_of_nonneg {f : ℕ → ℝ} (hf_nonneg : ∀ n, 0 ≤ f n) (hf : summable f) :
```

Comment:
The setting of the lemma is not the right one: you're writing it for functions on nat,  but the lemma is true in general.

Reply:
Indeed. And when I do it properly by using the definition of has_sum directly I don't need other lemmas in ennreal.lean and I can put that result where it belongs, in nnreal.lean.